adoc/MAIN.release-notes.adoc

```diff
@@ -60,11 +60,20 @@ To do this, enter your {tf} files/state folder and:
 * Install the latest version of {tf} using `zypper in terraform` (the installed version should be 0.12.19)
 * Navigate to your {tf} root folder (e.g. `/usr/share/caasp/terraform/vmware`)
 * Migrate {tf} files with the automatic migration tool by running `terraform 0.12upgrade`.
-** For OpenStack, run the extra operations for in-place upgrade, which follow just below.
-** For VMware, there is no extra operation but please note <<vmware_precaution_nullstrings>>.
+** For OpenStack, run the <<terraform-openstack-extra-ops>>
+** For VMware, run `terraform apply` to update the {tf} definitions to the new format used by 0.12.
```

Comment:
I would make this a generic comment for all platforms, I think it's a good recommendation to suggest a `terraform apply` with the latest 0.12, to make sure that the `tfstate` is up to date.

Reply:
Done